client/app/controllers/projects.js

```diff
@@ -0,0 +1,28 @@
+import Controller from '@ember/controller';
+
+export default class ProjectsController extends Controller {
+  // projects for applicant to do
+  get applicantProjects () {
+    // handle filtering here, reference in template as this.applicantProjects
+    // todo if statuscode is "Save" or "Package Preparation"
+    return this.model.filter((project) => project.packages.some((projectpackage) => {
+      if (projectpackage.statuscode === 'Saved' || projectpackage.statuscode === 'Package Preparation') {
+        return true;
+      } return false;
+    }));
+  }
+
+  // projects for nyc planning to do
+  get planningProjects () {
+    return this.model.filter((project) => project.packages.some((projectpackage) => {
+      if (projectpackage.statuscode === 'Submitted' || projectpackage.statuscode === 'Under Review' || projectpackage.statuscode === 'Revision Required') {
+        return true;
+      } return false;
+    }));
+  }
+
+  get upcomingProjects () {
```

Comment:
is 'upcomingProjects' the right name for no-PAS projects? 
I thought no-PAS projects go under the right-hand column, along with Planning Projects?

Maybe this should be renamed noPackages, like in the test 
https://github.com/NYCPlanning/labs-applicant-portal/pull/43/files#diff-0c14fdc9ee40bfbc92de4f1f8ec775ecR23

Reply:
@bfreeds you wanna address these?

Reply:
@godfreyyeung "upcomingPackages" naming would match the Figma wireframe, but I agree naming more descriptively to reflect what the logic of the data makes more sense in this case, rather than trying to reflect the eventual placement in the UI

Reply:
Fixed in new branch